easybuild/tools/options.py

```diff
@@ -813,6 +821,29 @@ def _postprocess_optarch(self):
             else:
                 self.log.info("Keeping optarch raw: %s", self.options.optarch)
 
+    def _postprocess_list_prs(self):
+        """Postprocess --list-prs options"""
+        list_prs_parts = self.options.list_prs.split(':')
+        nparts = len(list_prs_parts)
+
+        if nparts == 0:
+            list_prs_parts = [GITHUB_LIST_PR_STATES[0], GITHUB_LIST_PR_ORDERS[0], GITHUB_LIST_PR_DIRECTIONS[1]]
+        elif nparts == 1:
+            list_prs_parts.extend([GITHUB_LIST_PR_ORDERS[0], GITHUB_LIST_PR_DIRECTIONS[1]])
+        elif nparts == 2:
+            list_prs_parts.extend([GITHUB_LIST_PR_DIRECTIONS[1]])
+        elif nparts > 3:
+            raise EasyBuildError("Argument to --list-prs must be in the format state:order:direction")
+
+        if list_prs_parts[0] not in GITHUB_LIST_PR_STATES:
+            raise EasyBuildError("First element of --list-prs must be on of %s" % GITHUB_LIST_PR_STATES)
+        if list_prs_parts[1] not in GITHUB_LIST_PR_ORDERS:
+            raise EasyBuildError("Second element of --list-prs must be on of %s" % GITHUB_LIST_PR_ORDERS)
+        if list_prs_parts[2] not in GITHUB_LIST_PR_DIRECTIONS:
+            raise EasyBuildError("Third element of --list-prs must be on of %s" % GITHUB_LIST_PR_DIRECTIONS)
+
+        self.options.list_prs = list_prs_parts
```

Comment:
@migueldiascosta Rather than using `list_prs_parts`, I would create dedicated local constants, and then compose those at the end? And I would use a 3-tuple rather than a list (since it doesn't make sense to append more values to it, should always be exactly three).

```
self.options.list_prs = (list_pr_state, list_pr_order, list_pr_direc)